_data/projects/OpenRCT2.yml

```
@@ -9,8 +9,8 @@ tags:
 - rollercoasters
 - 2d
 upforgrabs:
-  name: good first issue
-  link: https://github.com/OpenRCT2/OpenRCT2/labels/good%20first%20issue
+  name: Hacktoberfest
```

Sadly these labels are case-sensitive when querying through the API, so they need to match

```suggestion
  name: hacktoberfest
```